install/scripts/cli.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+"""
+Hyperdome
+
+Copyright (C) 2019 Skyelar Craver <scravers@protonmail.com>
+                   and Steven Pitts <makusu2@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import click
+import sys
+import os
+import json
+from cryptography.hazmat.primitives.asymmetric.ed448 import Ed448PublicKey
+from cryptography.hazmat.primitives.serialization import (
+    load_ssh_public_key,
+    load_der_public_key,
+    load_pem_public_key,
+)
+
+
+version = "0.2"
+
+
+@click.group(invoke_without_command=True)
+@click.option("--debug", "-d", is_flag=True)
+@click.version_option(version, prog_name="Hyperdome Server")
+@click.pass_context
+def admin(ctx, debug):
+    if ctx.invoked_subcommand is not None:
+        return
+    if debug:
+        # TODO there must be a cleaner way to do this
+        sys.path.insert(
+            0,
+            os.path.dirname(
+                os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+            ),
+        )
+        sys.onionshare_dev_mode = True
+    import hyperdome_server
+
+    hyperdome_server.main()
+
+
+@admin.command()
+@click.option(
+    "--import",
+    "-i",
+    "import_",
+    type=click.File(),
+    multiple=True,
+    help="import public key from a file (filename used as NAME)",
```

Comment:
idk what "filename used as NAME" means

Reply:
as in the filename is taken to be the name of the counselor, e.g. carpenter.pub -> a counselor with username carpenter, I'll try to think of a clearer way to say that next PR.